### ISSUE TITLE
Bug in sortable lists official example

### ISSUE THREAD

**Issue by archit-p** [react-dnd/react-dnd#3212]
**Describe the bug**
Sortable lists example from the docs doesn't work as expected. The element being hovered on is moved instead of the element being dragged.

**Reproduction**
[Official CodeSandbox](https://codesandbox.io/s/github/react-dnd/react-dnd/tree/gh-pages/examples_hooks_js/04-sortable/simple?from-embed)

1. Go to the above sandbox.
2. Drag any of the cards.

**Expected behavior**
The element being dragged should be moved and not the element being hovered.

**Screenshots**
Notice that Write a JS Library card is being dragged, Write a README card is moving.
![react-dnd-bug](https://user-images.githubusercontent.com/30985772/119230664-e508f980-bb3a-11eb-9de6-dd338de26117.gif)

**Desktop (please complete the following information):**
Browser - firefox v88.0.1

**Comment by joshpetit** [comment]
Yea, we had the same problem in #3206, reproduced on Firefox and Chrome.

**Comment by timhooker** [comment]
We are seeing this problem as well but in a production environment with a similar implementation. Noteably we have a sortable list where the drop target nested inside the drag source. I can confirm I'm seeing this same behavior on the demo in Safari as well. Brave, for whatever reason, seems to exhibit errors less often than the other browsers I've checked.

**Comment by hakushi** [comment]
I'm getting the exact same behaviour. Any ideas on what could be the cause of this would be greatly appreciated.

**Comment by AndreBastos23** [comment]
Yeah i think we're having the same issue. Any fixes?

**Comment by joshpetit** [comment]
@AndreBastos23 Doesn't seem to be any besides using another library (or making a fork with a fix). There are a few other good replacement libraries, we moved over to [react-movable](https://www.npmjs.com/package/react-movable). Only disadvantage we found was not being able to [disable moves easily](https://github.com/tajo/react-movable/issues/49).

**Comment by hakushi** [comment]
Yeah, we ended up going with [react-beautiful-dnd](https://github.com/atlassian/react-beautiful-dnd) instead. Less code and better looking dnd out of the box as well.

**Comment by Nealyang** [comment]
i think we're having the same issue. Any fixes?
